apps/charterafrica/src/lib/ecosystem/utils.js

```diff
@@ -0,0 +1,24 @@
+export function localizeData(data = {}) {
```

Comment:
Where is this file being used?

Reply:
It is anticipated to be used in `ecosystem`.

Reply:
payload api does not allow create/update with data in the form 
```js
{
   description: {
    en: "English",
    pt: "Portuguese",
    fr: "French"
   }
}
```
Data should look like

```js
 createCollection(collection, toCreate, { locale })
```
So this function will be used to structure data in this format @kilemensi